src/microbit/test/test_button.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from ..model.button import Button
+
+
+class TestButton(object):
+    def setup_method(self):
+        self.button = Button()
+
+    @pytest.mark.parametrize("pressed", [True, False])
+    def test_is_pressed(self, pressed):
+        self.button._Button__pressed = pressed
+        assert pressed == self.button.is_pressed()
+
+    @pytest.mark.parametrize("was_pressed", [True, False])
+    def test_was_pressed(self, was_pressed):
+        self.button._Button__prev_pressed = was_pressed
+        assert was_pressed == self.button.was_pressed()
+        # Button resets prev pressed after was_pressed() is called
+        assert not self.button.was_pressed()
+
+    @pytest.mark.parametrize("presses", [0, 2, 4])
+    def test_get_presses(self, presses):
+        self.button._Button__presses = presses
+        assert presses == self.button.get_presses()
+        # Presses is reset to 0 after get_presses() is called
+        assert 0 == self.button.get_presses()

```

Comment:
I updated the tests. Let me know what you think. I originally had it that way because so it wouldn't rely on __press_down and __release in case those functions were broken.